platforms/js/build_js.py

```diff
@@ -161,6 +161,7 @@ def get_cmake_cmd(self):
             cmd.append("-DBUILD_WASM_INTRIN_TESTS=ON")
         else:
             cmd.append("-DBUILD_WASM_INTRIN_TESTS=OFF")
+        cmd.append("-DWITH_WEBGPU=ON")
```

Comment:
Please add an option for webgpu build. The default build should be without webgpu support.